### DISCUSSION TITLE
Setting up custom album art works... with an exception

### DISCUSSION THREAD

**Discussion by zafkiel-P** [jojo2357/Music-Discord-Rich-Presence#6] [Q&A] [answered]
So, I figured out how to set up the album artwork for DRP. However, coincidentally, one of my songs in Groove with a non-latin title actually contains an '=' in the name. As such, when I try to write the line in the .dat file, <album name containing equal sign>=<latinised title same as image name>, it doesn't work, as I assume DRP ignores everything past the first equal sign. Any ways to solve this, or is this something I just have to live with?

**Comment by jojo2357** [comment, marked as answer]
I will probably make it so that you just exclude the `=` since its the easiest fix and i doubt it will cause many big issues

**Comment by jojo2357** [comment]
Ooooooh this is a discussion. Please make an issue so i can close it when I fix it real fast xD

**Reply by zafkiel-P** [reply]
oh, sorry. I'm new to git, so wasn't sure where to put this. In the future, I'll put any issues in issues *facepalms*
Thanks for fixing it so quickly! I'll also make a new issue for you to close
